CRITICAL - 2022-10-09 00:43:58 --> Undefined variable $usuariosDb
in APPPATH\Views\vista_compras\frm_actualizar_pro.php on line 28.
 1 APPPATH\Views\vista_compras\frm_actualizar_pro.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usuariosDb', 'APPPATH\\Views\\vista_compras\\frm_actualizar_pro.php', 28)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\frm_actualizar_pro.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/frm_actualizar_pro', [], true)
 5 APPPATH\Controllers\Proveedores.php(44): view('vista_compras/frm_actualizar_pro', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Proveedores->datosProveedores('2')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Proveedores))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 01:15:57 --> syntax error, unexpected token ";"
in APPPATH\Controllers\Clientes.php on line 8.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Clientes.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Clientes')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Clientes')
 4 SYSTEMPATH\CodeIgniter.php(848): class_exists('\\App\\Controllers\\Clientes', true)
 5 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 01:16:23 --> syntax error, unexpected token ";"
in APPPATH\Controllers\Clientes.php on line 8.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Clientes.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Clientes')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Clientes')
 4 SYSTEMPATH\CodeIgniter.php(848): class_exists('\\App\\Controllers\\Clientes', true)
 5 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 01:17:39 --> Invalid file: vista_ventas/lista_cleintes.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('vista_ventas/lista_cleintes.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_ventas/lista_cleintes', [], true)
 3 APPPATH\Controllers\Clientes.php(11): view('vista_ventas/lista_cleintes', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Clientes->listaClientes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 01:17:42 --> Invalid file: vista_ventas/lista_cleintes.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('vista_ventas/lista_cleintes.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_ventas/lista_cleintes', [], true)
 3 APPPATH\Controllers\Clientes.php(11): view('vista_ventas/lista_cleintes', [...])
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Clientes->listaClientes()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 01:35:59 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Clientes.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Clientes->agregarClientes()
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#10 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-10-09 01:42:33 --> Undefined variable $pclientesDb
in APPPATH\Views\vista_ventas\frm_actualizar_cli.php on line 44.
 1 APPPATH\Views\vista_ventas\frm_actualizar_cli.php(44): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pclientesDb', 'APPPATH\\Views\\vista_ventas\\frm_actualizar_cli.php', 44)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_ventas\\frm_actualizar_cli.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_ventas/frm_actualizar_cli', [], true)
 5 APPPATH\Controllers\Clientes.php(44): view('vista_ventas/frm_actualizar_cli', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Clientes->datosClientes('1')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 01:44:04 --> Undefined variable $pclientesDb
in APPPATH\Views\vista_ventas\frm_actualizar_cli.php on line 44.
 1 APPPATH\Views\vista_ventas\frm_actualizar_cli.php(44): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pclientesDb', 'APPPATH\\Views\\vista_ventas\\frm_actualizar_cli.php', 44)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_ventas\\frm_actualizar_cli.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_ventas/frm_actualizar_cli', [], true)
 5 APPPATH\Controllers\Clientes.php(44): view('vista_ventas/frm_actualizar_cli', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Clientes->datosClientes('2')
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:23:08 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:23:08 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:23:29 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:23:29 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:24:35 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:24:35 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:25:51 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:25:51 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:26:05 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:26:05 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:27:35 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:27:35 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 08:28:37 --> mysqli_sql_exception: Table 'proy_pos.lista_empleados' doesn't exist in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-10-09 08:28:37 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 223.
 1 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\Empleados.php(12): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Empleados->listarEmpleados()
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:01:22 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:02:36 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:03:43 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:05:46 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:05:58 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:07:38 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:07:57 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:09:46 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:10:33 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:10:46 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:10:48 --> Class "App\Models\Lista_compra" not found
in APPPATH\Controllers\Compras.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:12:05 --> Undefined array key "idcompra"
in APPPATH\Views\vista_compras\lista_compras.php on line 60.
 1 APPPATH\Views\vista_compras\lista_compras.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "idcompra"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 60)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:13:59 --> Undefined array key "descripcion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:17:27 --> Undefined array key "descripcion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:17:56 --> Undefined array key "descripcion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:18:05 --> Undefined array key "descripcion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:18:17 --> Undefined array key "nombre_proveedor"
in APPPATH\Views\vista_compras\lista_compras.php on line 67.
 1 APPPATH\Views\vista_compras\lista_compras.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre_proveedor"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 67)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:19:02 --> Undefined array key "nombre_proveedor"
in APPPATH\Views\vista_compras\lista_compras.php on line 63.
 1 APPPATH\Views\vista_compras\lista_compras.php(63): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre_proveedor"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 63)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:19:25 --> Undefined array key "descripcion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-09 10:19:30 --> Undefined array key "descripccion"
in APPPATH\Views\vista_compras\lista_compras.php on line 62.
 1 APPPATH\Views\vista_compras\lista_compras.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripccion"', 'APPPATH\\Views\\vista_compras\\lista_compras.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\vista_compras\\lista_compras.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('vista_compras/lista_compras', [], true)
 5 APPPATH\Controllers\Compras.php(13): view('vista_compras/lista_compras', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->listarCompras()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 10:59:02 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Usuarios.php(22): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Usuarios->agregarUsuarios()
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-10-09 11:00:24 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Usuarios.php(22): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Usuarios->agregarUsuarios()
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-10-09 11:13:20 --> Cannot use object of type App\Models\Inventario as array
in APPPATH\Controllers\Compras.php on line 44.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->agregarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 11:16:50 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Compras.php(40): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Compras->agregarCompras()
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
#10 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-10-09 11:16:50 --> Cannot use object of type App\Models\Inventario as array
in APPPATH\Controllers\Compras.php on line 44.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->agregarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-09 11:18:11 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\proy_pos_pruebas\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\proy_pos_pruebas\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proy_pos_pruebas\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proy_pos_pruebas\system\Model.php(666): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proy_pos_pruebas\app\Controllers\Compras.php(40): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(898): App\Controllers\Compras->agregarCompras()
#9 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
#10 C:\xampp\htdocs\proy_pos_pruebas\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proy_pos_pruebas\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-10-09 11:18:11 --> Cannot use object of type App\Models\Inventario as array
in APPPATH\Controllers\Compras.php on line 44.
 1 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Compras->agregarCompras()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
